packages/connection-form/src/hooks/use-connect-form.ts

```diff
@@ -187,7 +201,9 @@ function buildStateFromConnectionInfo(
       : validateConnectionOptionsWarnings(
           initialConnectionInfo.connectionOptions
         ),
-    connectionOptions: cloneDeep(initialConnectionInfo.connectionOptions),
+    connectionOptions: adjustCSFLEParams(
+      initialConnectionInfo.connectionOptions
```

Comment:
Down the line will we remove this `adjustCSFLEParams` function? How would you feel about keeping the `cloneDeep` here as well as in that function to be sure we don't accidentally remove the cloning when we make that change. Probably a non-issue just thinking to avoid a slim chance at a possible bug way down the line.
```suggestion
      cloneDeep(initialConnectionInfo.connectionOptions)
```

Reply:
Sure, makes sense. Done!